Introduction.Rmd

```diff
@@ -125,76 +117,129 @@ The final three chapters describe general best practices not specifically tied t
 
 This is a lot to learn, but don't feel overwhelmed. Start with a minimal subset of useful features (e.g. just an `R/` directory!) and build up over time. To paraphrase the Zen monk Shunryu Suzuki: "Each package is perfect the way it is --- and it can use a little improvement".
 
-## Getting started {#intro-get}
+## devtools, usethis, and you {#intro-usage}
 
-To get started, make sure you have the latest version of R (at least `r paste0(version$major, ".", version$minor)`, which is the version that the code in this book uses), then run the following code to get the packages you'll need:
+> "I am large, I contain multitudes."
+>
+> --- Walt Whitman, Song of Myself
```

Comment:
Oh man, I have a strong aversion to Walt Whitman due to being forced to read his work during Honors English 11, my least favorite class taught by the teach I liked the least during High School. Seeing his name, and song of myself in particular, brought back bad memories.

Reply:
I thought I should actually read it, since I'm quoting, but I had no idea it was so long!